Changes in cps/md/midway/neogeo/nes/sms...

IcDasNMs opened the pull request:
2026/07/20

fightcade misc:
fixed corrupted .flt template, changed order in menu "input" [i_arabe]
added overclock option (available only for offline play) [requested by AliceMSU]
umk3 should be run at 54fps
new replayID.
Save presets without dipsw settings [dinkc64], backported to fightcade2 [IcDasNMs]
ridhero - Init [iq_132], fixed AES mode.
source: removed 'volume' from buttons list. [dinkc64]
Dinput, avoid checking keycode 0 - possible fix for spurious input [dinkc64]
Coin/Start padronization [dinkc64]: d_mitchell.cpp games, d_seta.cpp games, 1945kiii (now, inc. p2 coin), wwfwfest, snowbro2.
neogeo: code reordened + rom templates [taoenwen in fbneo repo].


d_cps1
added: cawingjr, sf2cepra, sf2cre
source: fixed dips in `sf2cej*` / sf2hfj. new mapper for `area88r`  [JacKc029735]

d_cps2
added: armwarb, dstlkb, mmancp2ur2, mmatrixa, mpanga, ringdestb, sfz3mix (0.31), sfzalr1 vsavb, xmvsfjr4
moved data: vsavae
updated: sf2prime, xmvsfj (possible V4), xmvsfjr1, xmvsfjr2, xmvsfjr3
source: support for sfz3mix 0.3x [barbudreadmon, dinkc64] -> backported for fc2 without samples, as requested by author

d_deco32
fixed tattass sound [dinkc64]

d_megadrive
added: md_2cnfnc, md_alicesisters, md_cannonfire, md_clanofheroes, md_handyharvy, md_ikplusdx, md_issdxnm (NTSC/60Hz mod), md_lwedding, md_mkae, md_mkaee, md_mk2wuec, md_popnwizz, md_drrobotndx, md_stonprotpiko, md_umk3ta (r5125, old umk3to, renamed) [gab75]
updated: md_cavestory, md_ffightmd, md_puzzulswap, md_umk3t (r5149, labels), md_wmargin (labels)

d_midway
added: umk3w, umk3wp

d_neogeo
added blueandr, crswd2blp, doubled1, mslugxmdj, mslug3hmy, mslug4zzf, mslug5tmj, sgz, turfmastsc, violentv, zintrckbp
updated: kf2k1ult, kof2000ps2, kof95t, kof98mix, kof99rp2f, mslugdyf2, rbff2bs, mslugxsc, mslugxcqi, mslug3cqi, mslug3cqtfb, mslug4zjhl, mslug5cq, samsho2pe
moved data: karnovrbh
source: overclock for mslugfc2, RAM-hack fix [FBNeo repository]

d_nes
added: nes_armwrestl, nes_bbbabes, nes_biocats, nes_blazrang, nes_catmercs, nes_clashcourts, nes_colormatcb, nes_contrahta, nes_contrahtn, nes_dataman, nes_famdepon, nes_firewall, nes_hbcpball, nes_hurdles, nes_justiceduela, nes_lightshields, nes_megakaijubb, nes_perfectdiamond, nes_spacegulls, nes_superstrife
updated: nes_almosthero, nes_justiceduel (labels) / nes_supertiltbro

d_sms
added: sms_4lexkiddnw, sms_alexkiddee, sms_antsms, sms_evil, sms_gotrisfv, sms_stella
updated: sms_alexkiddb

IcDasNMs committed:
changes until 22-06-2025

d_neogeo:
code moved karnovbh

d_cps2:
code moved vsavae (is normal rom, not phoenix)

d_megadrive:
updated umk3osc v31

IcDasNMs committed:
typo

IcDasNMs committed:
typo2

IcDasNMs committed:
removed flapchck bpanicdx

IcDasNMs committed:
osc typo

IcDasNMs committed:
karnovrbh correct name

IcDasNMs committed:
some changes in ng/cps

sfz3mix commented:
If it's not too much to ask, update (d_cps2) sf2prime to version 0.57 (relevant bug fix)

IcDasNMs committed:
updates until 28/06/2025

neogeo source: RAM hack fix
d_neogeo: updated mslug3cqi
d_cps2: updated sf2prime
d_megadrive: added squidgame
d_sms: added alexkiddee

IcDasNMs committed:
changes until 2025-07-05

d_cps2
updated sf2prime (v0.60 v2)

d_megadrive
added cannonfire, clanofheroes, handyharvy

IcDasNMs committed:
updates until - 2025-07-12

d_megadrive
added: alicesisters, ikplusdx, lwedding, stonprotpiko, umk3to (r5125)
labels: umk3t (r5149), wmargin

d_neogeo
updated: mslugdyf2, mslugxcqi, mslug3cqi, mslug3cqtfb, mslug4zjhl, mslug5cq

d_nes
added: armwrestl, bbbabes, biocats, blazrang, catmercs, lightshields, spacegulls
labels: almosthero

IcDasNMs committed:
updates until 2025-07-19

d_megadrive
updates: umk3t, umk3to (labels)

d_neogeo
updates: kf99rp2f, mslugdyf2

d_sms
added: stella

IcDasNMs committed:
megadrive - ffightmd

v087b

IcDasNMs committed:
updates until 2025-07-26

updates until 2025-07-26

d_megadrive
updated: ffightmd
labels: sor2upw

d_neogeo
update: kof98mix, mslugdyf2, mslugxcqi, mslug3cqi, mslug4zjhl

d_nes
added: dataman, famdepon, hbcpball
labels: tsbtapmaster

d_sms
updated: alexkiddb
labels: alexkiddee
added: antsms, evil, gotrisfv, linerunner

IcDasNMs committed:
updates - nes

system description: all games
nes
updated: miscastle

IcDasNMs committed:
updates until 2025-08-10

d_cps2
updates: sf2prime 0.63

d_megadrive
added uxmen
updates: umk3osc
source: system description for all games.

IcDasNMs committed:
Update d_megadrive.cpp

d_megadrive
renamed: umk3to to umk3ta [gab75]
labels: mkr

EShahverdian commented on src/burn/drv/megadrive/d_megadrive.cpp:
```diff
@@ -44237,20 +44237,21 @@ struct BurnDriver BurnDrvmd_telehugs = {
+STD_ROM_PICK(md_umk3ta)
+STD_ROM_FN(md_umk3ta)
 
-struct BurnDriver BurnDrvmd_umk3to = {
```
Is there a reason why struct BurnDriver BurnDrvmd_umk3to was changed to BurnDrvmd_umk3ta on lines 44248 - 44253?
 
 I'm just reviewing some code to learn and love fightcade 2.

IcDasNMs committed:
updates until 2025-08-23

d_cps1
updated sf2mix (+ dip switches) [JacKc]

d_cps2
updated ddsomjr1 ddsomjr2 [MAME team]

d_megadrive
updated cavestory

d_nes
added contrahta, contrahtn [AliceMSU]

IcDasNMs committed:
changes until 2025-08-30

d_megadrive
updated: cavestory

d_nes
added: perfectdiamond

IcDasNMs committed:
changes until 2025-09-06

d_cps2
updated: sf2prime [JacKc029735]

d_megadrive
updated: cavestory

d_nes
added: hurdles [gab75]

IcDasNMs committed:
changes until 2025-09-20

d_megadrive:
renamed: knnsf -> knnsfb (final version not supported yet)
updated: ffightmd
added: 2cnfnc

IcDasNMs committed:
updates until 2025-10-11

d_cps1:
Fixed 2 players mode DIP switch for sf2cejx and sf2hfj [JacKc029735]
added: sf2cre [dinkc64]

IcDasNMs committed:
changes until 2025-11-01

d_megadrive:
updated ffightmd

d_wunit:
added: umk3w, umk3wp (untested ROM, private?)

IcDasNMs committed:
changes until 2025-11-08

d_cps1
update: sf2mix

IcDasNMs committed:
changes until 2025-12-06

d_cps2
source: renamed burnDriver for: mmatrix, mmatrixj

d_megadrive
updated: bssf2gq

d_neogeo
updated: mslugxsc, mslugxcqi, mslug3cqi, mslug4zjhl, mslug5cq
added: mslugxmdj, mslug3hmy, mslug4zzf, mslug5tmj

IcDasNMs committed:
changes until 2025-12-14

2025-12-14

d_cps2
added: armwarb mmancp2ur2 mmatrixa mpanga ringdestb sfzalr1 vsavb xmvsfjr4
updated xmvsfj (possible V4) xmvsfjr1 xmvsfjr2 xmvsfjr3

d_megadrive
added: md_issdxnm (NTSC/60Hz mod)
source: updated forced-ds list

IcDasNMs committed:
changes until 2025-12-20

d_megadrive
added mkaee, mk2wuec [gab75]
update mk2wu (labels)

d_neogeo
added doubled1 [JacKc029735], violentv [Igor-Arabe]
update kf2k1ult [taoenwen]

IcDasNMs committed:
changes until 2025-12-24

source - app.rc
Fixed corrupted .flt template, changed order in menu "input" [i_arabe]

IcDasNMs committed:
changes until 2026-01-04

d_megadrive
update: ffightmd

IcDasNMs committed:
changes until 2026-01-17

d_cps1
added sf2cepra (training mode)

d_neogeo
updated violentv

IcDasNMs committed:
changes until 2026-02-01

d_cps2
updated sf2prime

d_megadrive
added mkae, popnwizz

d_neogeo
updated violentv

d_nes
added megakaijubb

IcDasNMs committed:
changes until 2026-02-07

d_nes
added: colormatcb, firewall, superstrife

IcDasNMs committed:
changes until 2026-02-15

cps2:
updated sf2prime [Igor-Arabe]

IcDasNMs committed:
changes until 2026-03-22

cps2
update: sf2prime 0.74

neogeo
removed flag: puzzldpr
labels: lastbld2eh, lastbld2te, mslug5w, magdrop3bh, rbff2bh, mslug5f, mslug5boss

IcDasNMs committed:
changes until 2026-04-04

d_cps2
updated: sf2prime

megadrive
updated: alicesisters

neogeo
updated: samsho2pe

IcDasNMs committed:
changes until 2026-04-10

megadrive
update: puzzulswap

neogeo
add: turfmastsc [bencao request]

IcDasNMs committed:
changes until 2026-04-14

add overclock menu as requested by [AliceMSU, Shine]
NB: available only in offline mode.

IcDasNMs committed:
changes - part II

d_neogeo
update: turfmastsc [requested by Bencao and an argentinean dude]

IcDasNMs committed:
changes until 2026-05-03

fightcade2
burn.h: new replayID
fbn_ggpo: umk3uc now runs at 54fps (online mode) [requested by lucasmarciel]

deco32: fixed tattass sounds [dinkc64], added replayid to check.

d_neogeo
updated: rbff2bs, kf2k1ult, turfmastsc

IcDasNMs committed:
changes until 2026-05-09

d_cps2
added: ddtoddp

d_megadrive
added: goldnax3de, bk3defpat, uxmen2

d_neogeo:
kof96pls labels

IcDasNMs committed:
changes until 2026-05-15

d_cps2
update: sf2prime

d_megadrive
add: sor2mmpr
labels: uxmen, uxmen2

IcDasNMs committed:
changes until 2026-05-24

d_megadrive
update: mkaee

d_neogeo
add: zintrckbp
update: samsho2pe
source: ridheroInit [iq_132], fixed AES mode.

Source
Save presets without dipsw settings [dinkc64], backported to fightcade2 [IcDasNMs]

IcDasNMs committed:
changes until 2026-05-29

d_neogeo
add: crswd2blp
update: samsho2pe
labels: zintrckbp

IcDasNMs committed:
changes until 2026-06-05

d_cps1
add cawingjr
code: area88r mapper [all by JacKc029735]

d_cps2
add dstlkb [JacKc029735]

source: removed 'volume' from buttons list. [dinkc64]
Dinput, avoid checking keycode 0 - possible fix for spurious input [dinkc64]
Coin/Start padronization [dinkc64]: d_mitchell.cpp games, 1945kiii (now, inc. p2 coin), wwfwfest, snowbro2.

IcDasNMs committed:
changes - some code moved

preparing to add new roms
and use #define ROMs [taoenwen]

IcDasNMs committed:
changes in d_megadrive...

removed accent in shaolincarcara

IcDasNMs committed:
changes - some code moved - part2

all games below teot was moved and added rom #define [taoenwen - fbneo repo].
next step: move all hacks above teot.

IcDasNMs committed:
change - d_neogeo code reordered

d_neogeo.cpp code totally reordered
1. Games in MAME
2. Games not on MAME
3. Bootlegs, hacks & decrypted sets {
 3.a bootlegs series
 3.b CD Conversion Series
 3.c Misc Hack Series
}
4. Diagnostic Cartridges
5. Demos and Homebrew

added #define <ROMS> [taoenwen, fbneo repo]

IcDasNMs committed:
changes until 2026-06-21

d_neogeo
updated: samsho2pe, kof95t, kof2000ps2
added: blueandr
src: names for some kof hacks / moved code for some mslug hacks

IcDasNMs committed:
changes until 2026-06-29

d_cps2
updated sfz3mix

d_neogeo
updated samsho2pe

fc2fbn-src
fixed dx9 in win8.1 [AliceMSU, fbneo repo]

IcDasNMs committed:
changes until 2026-07-12

d_cps2
updated: sf2prime [fbneo repo - Igor-Arabe]

d_megadrive
updated: mkaee [fbneo repo - Igor-Arabe]

d_neogeo
updated: zintrckbp [fbneo repo - Igor-Arabe]

d_seta.cpp: added 2p coin for stg, zingzip, oisipuzl [fbneo repo - barbudreadmon]

IcDasNMs committed:
changes until 2026-07-20

d_cps2
updated: sf2prime

d_megadrive
added drrobotndx

d_neogeo
added sgz
updated crswd2blp

IcDasNMs committed:
changes until 2026-07-27

d_cps1
code: move file definitions to cps.h [taoenwen]

d_cps2
update: sf2prime

d_neogeo
update: blueandr, turfmastsc
